Chaincase.UI/Pages/LoadWalletPage.razor

```diff
@@ -24,6 +24,14 @@
         If funds secured using these seed words are hidden behind a password you must enter it here.
         The password must be exactly as it was created. As it is your secret, we cannot check whether or not it is correct.
     </p>
+    <IonLabel class="ion-padding" id="blockheight">Optional: Block Height</IonLabel>
+    <IonItem>
+        <IonInput type="number" @bind-Value="@ViewModel.BlockHeight" 
+                  aria-labelledby="blockheight" min="@vm.MinBirthdayWallet.Value" />
+    </IonItem>
+     <p class="ion-padding">
+        If you know the block height of when this wallet was created, we will optimize the restoration to start from it.       
+     </p>
```

Comment:
I don't want to include new experience unless it's excellent so let's delete this.

1 q 1 suggestion: What happens if the actual birthday < user selected? Does it sync back? if up to date and 0 do we prompt a scan from 0? && We should use a calendar to pick a day rather than blockheight I don't speak blocktime

Reply:
silent ux failure sounds like panic to me. Let's not include